ldap/schema/02common.ldif

```diff
@@ -154,6 +154,7 @@ attributeTypes: ( 2.16.840.1.113730.3.1.2357 NAME 'nsTaskCurrentItem' DESC 'Slap
 attributeTypes: ( 2.16.840.1.113730.3.1.2358 NAME 'nsTaskTotalItems' DESC 'Slapi Task total items' SYNTAX 1.3.6.1.4.1.1466.115.121.1.27 SINGLE-VALUE X-ORIGIN '389 Directory Server' )
 attributeTypes: ( 2.16.840.1.113730.3.1.2359 NAME 'nsTaskCreated' DESC 'Slapi Task creation date' SYNTAX 1.3.6.1.4.1.1466.115.121.1.24 SINGLE-VALUE X-ORIGIN '389 Directory Server' )
 attributeTypes: ( 2.16.840.1.113730.3.1.2375 NAME 'nsTaskWarning' DESC 'Slapi Task warning code' SYNTAX 1.3.6.1.4.1.1466.115.121.1.27 SINGLE-VALUE X-ORIGIN '389 Directory Server' )
+attributeTypes: ( 2.16.840.1.113730.3.1.2385 NAME 'winSyncFlattenTree' DESC 'When set to on, will flatten tree structure in AD replication'  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE X-ORIGIN '389 Directory Server' )
 #
```

Comment:
OID is already in use in older branches (1.4.3)

     2.16.840.1.113730.3.1.2385     nsslapd-changelogcompactdb-time

It was removed in 1.4.4 when the changelog was moved in tothe main backend.  But this schema could be found during replication with mixed versions of DS.  

This instead can you use this oid:

    2.16.840.1.113730.3.1.2387